src/components/cards/NetworkCard/NetworkCard.vue

```diff
@@ -0,0 +1,44 @@
+<script setup lang="ts">
```

Comment:
I created this component as its own. It's used in the invest/create flows a lot so I thought might as well standardise it and have styles for it.